### ISSUE TITLE
"exclude" in ditaval results in ant BUILD FAILED

### ISSUE THREAD

**Issue by quipsy-karg** [dita-ot/dita-ot#1924]
I just migrated from DITA-OT 1.x to 2.0.1 and noticed that the same, unchanged project fails the ant build with BUILD FAILED result when there is an "exclude" in ditaval:
```
<val>
	<prop att="audience" val="developer" action="exclude"/>
</val>
```
As soon as I replace 'exclude' by either 'include' or 'flag', the result is BUILD SUCCESSFUL.

I assume that this is not my fault as the project runs well on DITA 1.x, so this looks like a bug.

**Comment by jelovirt** [comment]
Please link to a Gist with the verbose log output.

**Comment by quipsy-karg** [comment]
https://gist.github.com/quipsy-karg/b70ca43f22dd95bfc5ee

**Comment by jelovirt** [comment]
Seems to be a issue in PDF output. When particular elements are removed, the resulting FO file is invalid.

**Comment by quipsy-karg** [comment]
I hereby confirm that the solution provided by eerohele works like a charm even in my really complex filtering real-world environment. Thank You for providing this solution! :-)

@jelovirt Please merge to master! :-)

**Comment by robander** [comment]
Fixed with commit 0d73b27
